features/netsocket/emac-drivers/TARGET_NXP_EMAC/TARGET_IMX/imx_emac.cpp

```diff
@@ -189,7 +192,8 @@ bool Kinetis_EMAC::low_level_init_successful()
 
     /* Create buffers for each receive BD */
     for (i = 0; i < ENET_RX_RING_LEN; i++) {
-        rx_buff[i] = memory_manager->alloc_heap(ENET_ETH_MAX_FLEN, ENET_BUFF_ALIGNMENT);
+        rx_buff[i] = memory_manager->alloc_heap(ENET_ALIGN(ENET_ETH_MAX_FLEN, ENET_BUFF_ALIGNMENT),
```

Comment:
When I was inspecting it, I didn't think an extra `ENET_ALIGN` was necessary, as you're already passing in `ENET_BUFF_ALIGNMENT`, which should mean `alloc_heap` rounds up itself.

But because of the implementation, this will end up adding a couple more bytes, and maybe that makes the difference if we're overrunning?

Reply:
This analysis was incorrect, as per comment on #10239 - alloc_heap does not in fact round up the size.